README.md

```diff
@@ -132,13 +136,45 @@ Error message for required field
 - `min` : length of string
 - `confirmed `: confirmed passwords
 - `regex` : your custom regex
+- `required` : for custom required message
```

Comment:
mozno by sme modli pridat aj nejaky custom validate funkciu aby si mohol sam vlozit nejaky validator nieco ako:

https://vuelidate.netlify.com/#sub-simplest-example

Reply:
Added in 41869c9